src/auro-input.js

```diff
@@ -106,16 +114,140 @@ export default class AuroInput extends BaseInput {
     return null
   }
 
+  connectedCallback() {
+    super.connectedCallback();
+
+    if (this.format) {
+      var cleave = new Cleave(this, this.format);
+    }
+  }
+
+  getCreditCardIcon() {
+    return this.icon && this.format && this.format.hasOwnProperty('creditCard') ?
+      html`
+        <div class="creditCard-icon">
+         ${this.generateCreditCardIconHtml()}
```

Comment:
Would it be possible to also support the Alaska card logo for Alaska issued Visa Signature cards? Additionally, in the future, commercial accounts and e-certificate codes?

Reply:
The payment methods available on the Booking Payment page would be a great example of this

Reply:
Does it appear from the code that we are just looking for signature prefix numbers? I would assume that the Alaska card will have a signature set of numbers we can filter on? Cause yeah, we NEED to show the Alaska card icon. 

Good catch @gusnaughton

Reply:
Cards starting in "1" or "2" signifies a card associated with a "Major Industry Identifier (MII)" for Airlines. Digits 2-6 identify a particular institution.

From this I would gather it's likely that the AlaskaAir credit would have its on 2-6 digit identifier at the start of every card number.

https://www.creditcardinsider.com/learn/anatomy-of-a-credit-card/#credit-card-number

Adding another identifier to show a custom card for any identifier is trivial - given that unique identifier. @gusnaughton @blackfalcon either of you able to point me to where I can get that identifier value?

Reply:
Added the matched values for Alaska Air credit cards.